### ISSUE TITLE
IncrementalSource is undefined

### ISSUE THREAD

**Issue by mihailmilevfb** [DataDog/browser-sdk#1330]
I am working on an Ember application that has been successfully using `browser-rum` and `browser-logs` up to version 3.11.0. Today I tried to update to version 4.0.0 but I started seeing:

> Uncaught TypeError: Cannot read properties of undefined (reading 'IncrementalSource')

And the app just fails to load. Here is the exact line that triggers the error:

```
Object.defineProperty(exports, "IncrementalSource", { enumerable: true, get: function () { return record.IncrementalSource; } });
```

`@datadog/browser-rum` is being imported by using [ember-cli-cjs-transform](https://www.npmjs.com/package/ember-cli-cjs-transform) version 2.0.0.

Any ideas what might be going wrong?

**Comment by BenoitZugmeyer** [comment]
Could you make sure that all `@datadog/browser-*` packages are at the same version? (maybe check the lock file)
I'm not familiar with `ember-cli-cjs-transform`. Could you provide a minimal example so I can reproduce the issue?

**Comment by mihailmilevfb** [comment]
Hi Benoit :) Thanks for looking into the issue. It might be a false positive though. I did some things and here is what I was able to find out:
* Yes - all versions are the same in the lock file
* I tried setting up a barebones Ember project so I can reproduce the issue there and share it with you but I got a big surprise - everything is working properly in a fresh project

So what is different? The fresh Ember project is using `ember-auto-import` out of the box to make sure what's imported is also being bundled which is working fine. The app I am working on does not use it and is using some older methods to import the library (`ember-cli-cjs-transform`). If I switch to `ember-auto-import` to load browser-rum and browser-logs they get loaded there too!

So... Apparently there is something fishy in the app setup that I will keep on investigating. Sorry for bothering you. Hope this issue will be useful to anybody else - closing out :)
